packages/client/src/utils/helpers/handleDates.ts

```diff
@@ -0,0 +1,79 @@
+import moment, { Moment, unitOfTime } from 'moment'
+import { GroupBy } from '@cloud-carbon-footprint/common'
+import { ClientConfig } from 'src/Config'
+import loadConfig from 'src/ConfigLoader'
+import { ForecastDetails } from '../../pages/RecommendationsPage/RecommendationsTable/Forecast/Forecast'
+
+interface dateProps {
+  config?: ClientConfig
+}
+
+const groupByAmount = {
+  day: 30,
+  week: 4,
+  month: 1,
+}
+
+export const handleEmissionDateRange = ({
+  config = loadConfig(),
+}: dateProps) => {
+  const dateRangeType: string = config.DATE_RANGE.TYPE
+  const dateRangeValue: string = config.DATE_RANGE.VALUE
+
+  let endDate: moment.Moment = moment
+    .utc()
+    .subtract(config.MINIMAL_DATE_AGE, 'days')
+  if (config.END_DATE) {
+    endDate = moment.utc(config.END_DATE)
+  }
+
+  let startDate: moment.Moment
+  if (config.PREVIOUS_YEAR_OF_USAGE) {
+    startDate = moment.utc(Date.UTC(endDate.year() - 1, 0, 1, 0, 0, 0, 0))
+  } else if (config.START_DATE) {
+    startDate = moment.utc(config.START_DATE)
+  } else {
+    startDate = moment
+      .utc()
+      .subtract(dateRangeValue, dateRangeType as unitOfTime.DurationConstructor)
+  }
+
+  return {
+    start: startDate,
+    end: endDate,
+  }
+}
+
+export const checkFootprintDates = ({
+  footprint,
+  groupBy = 'day' as GroupBy,
```

Comment:
The default value can probably be simplified to the following: 
```suggestion
  groupBy = GroupyBy.day,
```

However, I'm wondering if we even need a default value or if that's something this function should be concerned with. Isn't the default groupBy set elsewhere?

Reply:
Adding type error here for context, there is probably a way to resolve this but might leave it for a separate fix:
ERROR in ../common/src/index.ts 10:7

Module parse failed: Unexpected token (10:7)
File was processed with these loaders:

```
 * ../../node_modules/@pmmmwh/react-refresh-webpack-plugin/loader/index.js
 * ../../node_modules/source-map-loader/dist/cjs.js
You may need an additional loader to handle the result of these loaders.
| export { default as configLoader, setConfig } from './ConfigLoader'
| export { default as Config } from './Config'
> export type { QUERY_DATE_TYPES, CCFConfig, AWSAccount } from './Config'
| export { GroupBy } from './Config'
| export {
```

Reply:
No worries, it's minor so we can leave it as-is